src/ctia/schemas/core.clj

```diff
@@ -316,3 +316,12 @@
 (defschema ObservableTypeIdentifier
   vocs/ObservableTypeIdentifier
   "vocab.observable-type-id")
+
+(s/defschema ConfigurationServices
```

Comment:
There might be a better name for this schema. I might call it `GetEntitiesServices`.